packages/schematics/angular/config/schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "SchematicsAngularConfig",
+  "title": "Angular Config File Options Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "Generates a configuration file in the given project.",
+  "properties": {
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "$default": {
+        "$source": "projectName"
+      }
+    },
+    "type": {
+      "type": "string",
```

Comment:
nit: Maybe also add the following to allow specifying the config type directly:
```
      "$default": {
        "$source": "argv",
        "index": 0
      },
```
Example: `ng g config karma`

Reply:
Agreed.